.circleci/config.yml

```diff
@@ -29,14 +29,13 @@ jobs:
       - run:
           name: Install dependencies (dash)
           command: |
-              git clone git@github.com:plotly/dash.git
-              git clone git@github.com:plotly/dash-renderer.git
+              git clone git@github.com:plotly/dash.git dash-main
```

Comment:
shallow clone can save a lot of time, no?

Reply:
sure, will search and replace for table/html/core